mkt/webapps/serializers.py

```diff
@@ -328,6 +338,14 @@ def save_upsold(self, obj, upsold):
             current_upsell.delete()
 
     def save_price(self, obj, price):
```

Comment:
Maybe add a comment explaining that this shouldn't be called for free apps, and the caller is responsible for checking this beforehand.